cmd/podman/images/push.go

```diff
@@ -224,6 +228,10 @@ func imagePush(cmd *cobra.Command, args []string) error {
 		}
 	}
 
+	if pushOptions.PushUntagged {
+		destination = destination + reference.UnknownDigestSuffix
```

Comment:
(I’m not sure this needs to exist; the user can just use the suffixed syntax directly. Ultimately, I think that will be the case for `farm`…)